### ISSUE TITLE
Add striped color option for wire

### ISSUE THREAD

**Issue by leannapancoast** [wokwi/wokwi-features#1039]
Could a striped color option be added to the wires? My immediate goal is to be able to do a black-and-white printout and let students still be able to tell the wires apart. A secondary goal would be higher accessibility option for those that are color blind.

Maybe also an option to change the thickness or border shape of the wires? Like a curvy or zig-zag option. Though those would be quite difficult to find in the wild.

**Comment by cgten-uk** [comment]
+1 for this.  There are limited colour options for wires, which means having to reuse colours, making diagrams hard to read.

I like to organise my wiring based on the component, but then differentiate each wire to that component.

For example...

For an RGB LED:
- All wires to the LED are yellow as the primary colour.
- Each channel is the respective light colour as the secondary.
- Common wire is just yellow.
- That means for a Yellow/Red wire, I know that's the red channel for the RGB LED, Yellow/Green is the green channel, etc...

For a potentiometer:
- All wires are blue as the primary, except GND which is black.
- DT is Blue/Purple.
- CLK is Blue/Green.
- SW is Blue/Yellow
- VCC is solid Blue.

Ideally the primary colour should have wider bands than the secondary, so it's easy to tell Blue/Yellow apart from Yellow/Blue.
